interface/themes/style_light.css

```diff
@@ -1492,3 +1492,8 @@ table.dataTable#therapy_groups_list tbody td{
   margin: 20px 20px 2px 20px;
 
 }
+
+.sidebar {
+  background: #f5f5f5;
+  border-right: 1px solid #eee;
+}
```

Comment:
The other styles will work without this, I just wanted to make my style look a bit better. This just won't look as good on the old themes, but that's because the old themes just don't look good